_component_design/specs-states.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Specs and states
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: |
+  Helix controls allow users to select options in a variety of ways including
```

Comment:
@araiford - Is this usage statement correct? I made a new file, seeing as this did not previously exist (unless it did under a different name?)